.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: feature-request
+assignees: ''
+
+---
+
+**Describe the feature or improvement**
+
+_A clear and concise description of the proposed feature or change. Eg. It's important for zkApp developers to prove circuits in <20s_  
+
+**Describe the solution**
+
+_A clear _technical_ description of the implementation._
```

Comment:
The solution may not be known for many features.  Should we have a different template for "wishlist" features and feature requests with a specific solution in mind?